### ISSUE TITLE
No loader is configured for .node

### ISSUE THREAD

**Issue by ermincelikovic** [jycouet/remult-kit#21]
✘ [ERROR] No loader is configured for ".node" files: ../../node_modules/.pnpm/@node-rs+argon2-darwin-arm64@1.7.0/node_modules/@node-rs/argon2-darwin-arm64/argon2.darwin-arm64.node

    ../../node_modules/.pnpm/@node-rs+argon2@1.7.0/node_modules/@node-rs/argon2/index.js:173:36:
      173 │ ...ire('@node-rs/argon2-darwin-ar...
          ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by GonerGenesis** [comment]
this is a vite issue ... for the time being you could exclude the lib in your `vite.config.ts` with
```
optimizeDeps: {
    exclude: [
      '@node-rs/argon2-darwin-arm64'
    ]
  }
```

**Comment by jycouet** [comment]
This should be automatically done with the latest vite `remultKit` plugin.
I keep the issue open for now to test it in different context.

**Comment by GonerGenesis** [comment]
I've got the issue with `vite: "^5.2.10", "remult-kit": "0.26.4-exp.2"` on  `'@node-rs/bcrypt-linux-x64-gnu',
      '@node-rs/argon2-linux-x64-musl',
      '@node-rs/argon2-linux-x64-gnu',
      '@node-rs/bcrypt-linux-x64-musl'`

vs code gives me this hint on `remultKit`:
```
No overload matches this call.
  The last overload gave the following error.
    Type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption'.
      Type 'Plugin<any>' is not assignable to type 'PluginOption'.
        Type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").Plugin<any>' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").Plugin<any>'.
          Types of property 'apply' are incompatible.
            Type '"serve" | "build" | ((this: void, config: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").ConfigEn...' is not assignable to type '"serve" | "build" | ((this: void, config: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").Config...'.
              Type '(this: void, config: UserConfig, env: ConfigEnv) => boolean' is not assignable to type '"serve" | "build" | ((this: void, config: UserConfig, env: ConfigEnv) => boolean) | undefined'.
                Type '(this: void, config: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").ConfigEnv) => boolean' is not assignable to type '(this: void, config: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").ConfigEnv) => boolean'.
                  Types of parameters 'config' and 'config' are incompatible.
                    Type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig'.
                      Types of property 'plugins' are incompatible.
                        Type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption[] | undefined' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption[] | undefined'.
                          Type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption[]' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption[]'.
                            Type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").PluginOption'.
                              Type 'Plugin<any>' is not assignable to type 'PluginOption'.
                                Type 'import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").Plugin<any>' is not assignable to type 'import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").Plugin<any>'.
                                  Types of property 'apply' are incompatible.
                                    Type '"serve" | "build" | ((this: void, config: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.2.10_@types+node@20.12.8/node_modules/vite/dist/node/index").Config...' is not assignable to type '"serve" | "build" | ((this: void, config: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").UserConfig, env: import(".../node_modules/.pnpm/vite@5.1.6_@types+node@20.12.8/node_modules/vite/dist/node/index").ConfigEn...'.
                                      Type '(this: void, config: UserConfig, env: ConfigEnv) => boolean' is not assignable to type '"serve" | "build" | ((this: void, config: UserConfig, env: ConfigEnv) => boolean) | undefined'.ts(2769)
index.d.ts(3154, 18): The last overload is declared here.
```
maybe it's the node version?

**Comment by jycouet** [comment]
Should be fixed in `remult-kit@0.26.4-exp.3`
